src/components/roadmapList/Pagination.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import style from './RoadmapList.module.css';
+
+import Icon from 'components/ui/Icon';
+
+interface Props {
+  totalPages?: number;
+  currentPage: number;
+  onPageChange: (value: number) => void;
+}
+
+const Pagination: React.FC<Props> = ({
+  totalPages = 10,
+  currentPage = 1,
+  onPageChange,
+}) => {
+    const getPageNumbers = () => {
+        const pageNumbers:number[] = [];
+        if (totalPages <= 3) {
+          for (let i = 1; i <= totalPages; i++) {
+            pageNumbers.push(i);
+          }
+        } else {
+          if (currentPage <= 2) {
+            for (let i = 1; i <= 3; i++) {
+              pageNumbers.push(i);
+            }
+          } else if (currentPage >= totalPages - 1) {
+            for (let i = totalPages - 2; i <= totalPages; i++) {
+              pageNumbers.push(i);
+            }
+          } else {
+            for (let i = currentPage - 1; i <= currentPage + 1; i++) {
+              pageNumbers.push(i);
+            }
+          }
+        }
+        return pageNumbers;
+      };
+
+  const handleIncrease =()=>{
+    if(currentPage === totalPages) return
+    onPageChange(currentPage + 1)
+
+  }
+
+  const handleDecrease =()=>{
+    if(currentPage === 1) return
+    onPageChange(currentPage - 1)
+  }
+
+  const handleNumberClick = (value:number)=>{
+    onPageChange(value)
+  }
+
+  return (
+    <div className={style.pagination}>
+      <button
+        onClick={handleDecrease}
+        disabled={currentPage === 1}
+        className={style.pagination_button}
+      >
+        <Icon id="icon-dropdown" fill="#2E3249" width={28} height={28} />
+      </button>
+
+      {getPageNumbers().map((number) => (
+        <button
+          key={number}
+          onClick={()=>handleNumberClick(number)}
+          className={`${style.pagination_button} ${style.pagination_number}`}
+        >
+          <span className={currentPage === number ? style.active_page : ''}>
+            {number}
+          </span>
+        </button>
+      ))}
+      <button
+        onClick={handleIncrease}
+        disabled={currentPage === totalPages}
+        className={style.pagination_button}
+      >
+        <Icon
+          id="icon-dropdown"
+          fill="#2E3249"
+          width={28}
+          height={28}
+          boxStyles={style.arroy_right}
```

Comment:
var, size again, and if I'm not mistaken you need to write "arrow"

Reply:
Corrected +